src/tools/tidy/src/extdeps.rs

```diff
@@ -6,6 +6,10 @@ use std::path::Path;
 /// List of whitelisted sources for packages.
 const WHITELISTED_SOURCES: &[&str] = &[
     "\"registry+https://github.com/rust-lang/crates.io-index\"",
+
+    "\"git+https://github.com/michaelwoerister/std-mangle-rs?\
```

Comment:
This should be remove in the final version.

Reply:
It should be, but maybe that change was lost in a rebase.